### DISCUSSION TITLE
Spam bots attack

### DISCUSSION THREAD

**Discussion by serge-rider** [dbeaver/dbeaver#29266] [General]
We are experiencing a massive spam attack. Spam bots create thousands of discussions and comments.

We clean them up regularly, and I hope we didn't delete any real discussions.
Please leave comments here if so.

Also, please share any ideas about spam bot protection.

**Comment by GaryAustin1** [comment]
You guys figure anything out to block this spam?  It is getting out of control on a forum I work with.

**Reply by serge-rider** [reply]
Unfortunately no. 
For now we can only ban bots one by one. And cleanup spam by semi-automatic tools. 

At the moment GitHub doesn't provide any good way to prevent spam. I am trying to communicate with them but with no no luck :(

**Comment by shoerley** [comment]
Just a feedback about spam bots reporting here. I've spotted two accounts that are spamming this repo, and found out that the accounts were especially created to do that. So I decided to report them to Github.

What a surprise to see that reporting a spam account is a bit of a struggle : solve enigmas, answers several questions, decribe why I report with a minimal character count ... Although it may seem legitimate, I found frustrating that it is much harder to report spam than to actually spam a discussion.

**Reply by serge-rider** [reply]
Yes, that's really frustrating. We banned tens of spam accounts too. Hopefully GitHub will add a smart spam protection one day..
Thank you )
